lntest/utils.go

```diff
@@ -164,8 +164,9 @@ func NodeArgsForCommitType(commitType lnrpc.CommitmentType) []string {
 func CalcStaticFee(c lnrpc.CommitmentType, numHTLCs int) btcutil.Amount {
 	const (
 		htlcWeight         = input.HTLCWeight
-		anchorSize         = 330
-		defaultSatPerVByte = lnwallet.DefaultAnchorsCommitMaxFeeRateSatPerVByte
+		anchorSize         = 330 * 2
+		defaultSatPerVByte = lnwallet.DefaultAnchorsCommitMaxFeeRateSatPerVByte //nolint:lll
```

Comment:
nit: add the comment above the `const` keyword or above the affected line to avoid making the line even longer?

Reply:
done!